plugins/mcreator-localization/lang/texts.properties

```diff
@@ -899,6 +899,7 @@ blockly.block.entity_checkgamemode=Is %1 in game mode
 blockly.block.entity_clear_inventory=Clear inventory of %1
 blockly.block.entity_clearpotions=Clear all potion effects of %1
 blockly.block.entity_close_gui=Close any GUI open for %1
+blockly.block.entity_create=%1 Create new %2 without spawning
```

Comment:
%1 Create new %2 instance without spawning

that would stress more new instance is made every time this is called

Reply:
Erm... So just add **instance** word here?

Reply:
Yep

Reply:
yeah